docs/source/docker_only.rst

```diff
@@ -0,0 +1,59 @@
+It is possible to develop with dallinger using only docker as a prerequisite.
+This guide goes through the necessary steps to achieve this.
+
+Set up services
+===============
+
+Start services needed by dallinger:
+
+.. code-block:: shell
+
+    docker run -d --name dallinger_redis --network=dallinger -v dallinger_redis:/data redis redis-server --appendonly yes
+    docker run -d --name dallinger_postgres --network=dallinger -e POSTGRES_USER=dallinger -e POSTGRES_PASSWORD=dallinger -e POSTGRES_DB=dallinger -v dallinger_postgres:/var/lib/postgresql/data postgres:12
+
+Start adminer:
+
+.. code-block:: shell
+
+    docker run -d --name dallinger_adminer --network dallinger --link dallinger_postgres:db adminer
```

Comment:
Missing argument?

```
-p 8000:8080
```

Reply:
I think I might be misunderstanding it actually.

Reply:
I'm conflicted about this.
If you want to access a service in a container you can either connect to the container IP address and service port, or expose the container port on localhost and connect to it. The former has one extra step to take care of (looking up the container IP address). The latter might not work in some cases (if port 8080 is for some reason already used by a service on the host).
I'm not sure which one is better to suggest in the docs.

Reply:
Will the container IP address always be the same, if for example we restart the computer? 

My main reason for commenting was that I didn't manage to access the database page at all using the current instructions, and this was the only way I found to make it work. I must have been doing something wrong but it might be worth double-checking the exact documentation.

Reply:
Thinking about it, I would say the latter approach is better if we are always going to run the command interactively, but in the future I suppose we would try and wrap this in a script, in which case the former would be better. Do you agree?

Reply:
A script might print the full URL to adminer to make things easier, for sure.
The preferred way should be the one that is less likely to cause confusion/issues, and I'm not in a good position to make a good assessment. Both ways look pretty straightforward to me. I went with this one because I thought that an occupied `8080` port on localhost might be a bit dififcult to troubleshoot, but given your feedback I now think it's probably better to suggest to bind port `8080` and use `localhost` in the URL.

Reply:
Happy with this if you're happy - it seems quite common to do things this way in tutorials for Docker services